lib/foreman_maintain/check.rb

```diff
@@ -15,18 +15,14 @@ class Check < Executable
     #                   the failure, will be offered to the user when running
     #                   in interactive mode
     #
-    # * +:warn* - issue warning instead of failure: this is less strict check,
-    #      that could be considered as non-critical for continuing with the scenario
+    # * +:error_type* - error type if wants to override failure
```

Comment:
@iNecas , @swapab 
Here, I have replaced `:warn` option to `:error_type`.
Using this option, we can raise warning or any other error_type of exceptions.